<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TalkGui</class>
  <widget class="QDialog" name="TalkGui">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>259</width>
        <height>218</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Form1</string>
    </property>
    <property name="toolTip" stdset="0">
      <string/>
    </property>
    <property name="whatsThis" stdset="0">
      <string>dialog





in this box you see the conversation with the player whose name appears on the tab label</string>
    </property>
    <layout class="QGridLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item row="2" column="0">
        <layout class="QHBoxLayout">
          <item>
            <widget class="QLineEdit" name="LineEdit1"/>
          </item>
          <item>
            <widget class="QPushButton" name="pb_match">
              <property name="maximumSize">
                <size>
                  <width>28</width>
                  <height>28</height>
                </size>
              </property>
              <property name="text">
                <string/>
              </property>
              <property name="icon">
                <iconset>image0</iconset>
              </property>
              <property name="toolTip" stdset="0">
                <string>match</string>
              </property>
              <property name="whatsThis" stdset="0">
                <string>match





proposes a match to this player</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pb_releaseTalkTab">
              <property name="maximumSize">
                <size>
                  <width>28</width>
                  <height>28</height>
                </size>
              </property>
              <property name="text">
                <string/>
              </property>
              <property name="icon">
                <iconset>image1</iconset>
              </property>
              <property name="toolTip" stdset="0">
                <string>close</string>
              </property>
              <property name="whatsThis" stdset="0">
                <string>close





closes the tab (and dialog)</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="1" column="0">
        <widget class="Q3TextEdit" name="MultiLineEdit1">
          <property name="readOnly">
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0">
            <string>dialog with this player</string>
          </property>
          <property name="whatsThis" stdset="0">
            <string>dialog





in this box you see the conversation with the player whose name appears on the tab label</string>
          </property>
        </widget>
      </item>
      <item row="0" column="0">
        <widget class="Q3Frame" name="stats_layout">
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
            <number>0</number>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>0</number>
            </property>
            <item row="0" column="0">
              <layout class="QHBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>0</number>
                </property>
                <item>
                  <layout class="QGridLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <property name="spacing">
                      <number>0</number>
                    </property>
                    <item row="0" column="2">
                      <widget class="QLabel" name="stats_rating">
                        <property name="sizePolicy">
                          <sizepolicy>
                            <hsizetype>3</hsizetype>
                            <vsizetype>5</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="font">
                          <font>
                            <bold>true</bold>
                          </font>
                        </property>
                        <property name="text">
                          <string>--</string>
                        </property>
                        <property name="alignment">
                          <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="2" column="1" rowspan="1" colspan="2">
                      <widget class="QLabel" name="stats_idle">
                        <property name="sizePolicy">
                          <sizepolicy>
                            <hsizetype>3</hsizetype>
                            <vsizetype>5</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="font">
                          <font>
                            <bold>true</bold>
                          </font>
                        </property>
                        <property name="text">
                          <string>--</string>
                        </property>
                        <property name="alignment">
                          <set>Qt::AlignCenter</set>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="2" column="0">
                      <widget class="QLabel" name="Label_Idle">
                        <property name="minimumSize">
                          <size>
                            <width>80</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>Idle :</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="0" rowspan="1" colspan="2">
                      <widget class="QLabel" name="textLabel1">
                        <property name="minimumSize">
                          <size>
                            <width>80</width>
                            <height>0</height>
                          </size>
                        </property>
                        <property name="maximumSize">
                          <size>
                            <width>80</width>
                            <height>32767</height>
                          </size>
                        </property>
                        <property name="text">
                          <string>Rating :</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="0" rowspan="1" colspan="3">
                      <layout class="QHBoxLayout">
                        <property name="margin">
                          <number>0</number>
                        </property>
                        <property name="spacing">
                          <number>0</number>
                        </property>
                        <item>
                          <widget class="QLabel" name="textLabel1_2">
                            <property name="minimumSize">
                              <size>
                                <width>80</width>
                                <height>0</height>
                              </size>
                            </property>
                            <property name="text">
                              <string>Rated games :</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QLabel" name="stats_rated">
                            <property name="sizePolicy">
                              <sizepolicy>
                                <hsizetype>3</hsizetype>
                                <vsizetype>5</vsizetype>
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                              </sizepolicy>
                            </property>
                            <property name="font">
                              <font>
                                <bold>true</bold>
                              </font>
                            </property>
                            <property name="text">
                              <string>--</string>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </item>
                <item>
                  <layout class="QVBoxLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <property name="spacing">
                      <number>0</number>
                    </property>
                    <item>
                      <layout class="QHBoxLayout">
                        <property name="margin">
                          <number>0</number>
                        </property>
                        <property name="spacing">
                          <number>0</number>
                        </property>
                        <item>
                          <widget class="QLabel" name="textLabel1_2_2_2">
                            <property name="text">
                              <string>Country :</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QLabel" name="stats_country">
                            <property name="sizePolicy">
                              <sizepolicy>
                                <hsizetype>3</hsizetype>
                                <vsizetype>5</vsizetype>
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                              </sizepolicy>
                            </property>
                            <property name="font">
                              <font>
                                <bold>true</bold>
                              </font>
                            </property>
                            <property name="text">
                              <string>--</string>
                            </property>
                            <property name="alignment">
                              <set>Qt::AlignCenter</set>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <layout class="QHBoxLayout">
                        <property name="margin">
                          <number>0</number>
                        </property>
                        <property name="spacing">
                          <number>0</number>
                        </property>
                        <item>
                          <layout class="QHBoxLayout">
                            <item>
                              <widget class="QLabel" name="textLabel1_3">
                                <property name="text">
                                  <string>Wins :</string>
                                </property>
                                <property name="wordWrap">
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QLabel" name="stats_wins">
                                <property name="font">
                                  <font>
                                    <bold>true</bold>
                                  </font>
                                </property>
                                <property name="text">
                                  <string>--</string>
                                </property>
                                <property name="alignment">
                                  <set>Qt::AlignVCenter|Qt::AlignLeft</set>
                                </property>
                                <property name="wordWrap">
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </item>
                        <item>
                          <layout class="QHBoxLayout">
                            <item>
                              <widget class="QLabel" name="textLabel1_3_2">
                                <property name="text">
                                  <string>Loss :</string>
                                </property>
                                <property name="wordWrap">
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QLabel" name="stats_loss">
                                <property name="font">
                                  <font>
                                    <bold>true</bold>
                                  </font>
                                </property>
                                <property name="text">
                                  <string>--</string>
                                </property>
                                <property name="alignment">
                                  <set>Qt::AlignVCenter|Qt::AlignLeft</set>
                                </property>
                                <property name="wordWrap">
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <layout class="QHBoxLayout">
                        <property name="margin">
                          <number>0</number>
                        </property>
                        <property name="spacing">
                          <number>0</number>
                        </property>
                        <item>
                          <widget class="QLabel" name="textLabel1_2_2_2_2_2">
                            <property name="text">
                              <string>playing game :</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QLabel" name="stats_playing">
                            <property name="font">
                              <font>
                                <bold>true</bold>
                              </font>
                            </property>
                            <property name="text">
                              <string>--</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item row="1" column="0">
              <layout class="QHBoxLayout">
                <item>
                  <widget class="QLabel" name="textLabel1_2_2">
                    <property name="minimumSize">
                      <size>
                        <width>92</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="text">
                      <string>Address :</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="stats_address">
                    <property name="font">
                      <font>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>--</string>
                    </property>
                    <property name="frame">
                      <bool>false</bool>
                    </property>
                    <property name="readOnly">
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="3" column="0">
              <widget class="QLabel" name="stats_info">
                <property name="text">
                  <string>--</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="stats_default">
                <property name="text">
                  <string>--</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <images>
    <image name="image0">
      <data format="PNG" length="417">89504e470d0a1a0a0000000d49484452000000100000001008060000001ff3ff610000016849444154388da5934baee23010458f3f8948400810167b610fcc1112dbcd5af80489499c60c7f69b74221e34dd7add77645955f7537689aaaa78c67ebf4f7ddfe3bde7f178e09cc37b4f080180aaaac473bd1e0ec7e3314929994c260078ef514a01106324c6484a89ed769b9e8924c0e17048524a8aa260b55a618cc118c362b1a02c4bb4d608f14d7824d283429665cce77396cb2565591242a0ae6b628c745d87738e1823afd0bbdd2e154581528aa22858afd718639052329d4e69db96fbfdfee66070a1fbbe2784408c9121c666b361369b916519a7d309adf55bf300e9bdc73947dbb6344d83b516ef3d5a6bf23c1f879852fa2d81f6de23844029455dd7e4798e1082ebf54a5dd75c2e17acb5e333be11841070ce9152228440d775dc6e37f23ca7691acee733d6da8f0e049084104829514aa1b51e330ff13ea98f04df2e9ea6fd49f58f043f857cfddb3f41555542fe8f3afcda857f7131f488d7751e96e46f8d03be007631c227f0d0fafa0000000049454e44ae426082</data>
    </image>
    <image name="image1">
      <data format="PNG" length="253">89504e470d0a1a0a0000000d4948445200000011000000100806000000f031945f000000c449444154388dc593bb11c4200c4497ab48b1720aa00d7257e1dc6da80072c574c445cb18836fee13dc86da9d272120b83b7ed5e367c23b10556daadafe33c9abce2b6f82e49cdb5d9835669610336bb5560080bb872b84b55a2bccac830648290500b06ddbdd69bac7ec04e11429a5690a8a1eb313e45b0d10110180e1bc57d1637682c4180100fbbef79abb87f392e9313b41524a811d54b59d2732b3fe724564d85b587dc09c733b2fee2c11c1711cc3e29710762ea5f45b1011c4189737770bf9444f081563aba4fa97130000000049454e44ae426082</data>
    </image>
  </images>
  <connections>
    <connection>
      <sender>pb_releaseTalkTab</sender>
      <signal>pressed()</signal>
      <receiver>TalkGui</receiver>
      <slot>slot_pbRelTab()</slot>
    </connection>
    <connection>
      <sender>pb_match</sender>
      <signal>pressed()</signal>
      <receiver>TalkGui</receiver>
      <slot>slot_match()</slot>
    </connection>
  </connections>
</ui>
